### ISSUE TITLE
Is it possible to only allow HTML for 1 page and disallow all others?

### ISSUE THREAD

**Issue by VNRARA** [saggi-dw/dokuwiki-plugin-htmlok#14]
Is it possible to only allow HTML for 1 page and disallow all others?

I want to add HTML to the homepage to make the landing better vs. the standard landing but don't want other people to use HTML. Is this possble?

**Comment by Betusto** [comment]
Yeah, also an option to disable html in specific page or block would be great, since some plugins behave odd when allowing html
